oca_port/squash_bot_commits.py

```diff
@@ -0,0 +1,154 @@
+import subprocess
+from .utils.misc import Output, bcolors as bc
+import click
+import tempfile
+import os
+from .utils import git as g
+
+
+MESSAGE_TO_SQUASH = [
+    "Added translation using Weblate",
+    "Translated using Weblate",
```

Comment:
These ones are genuine commits with human authors.
But it's true that if there are some sequential commits from same author for same language, they could be squashed.